packages/pipelines/src/hooks/usePipelineRuns.ts

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { PipelineRunKind } from '../types';
+import { GetNextPage, Selector } from '../types/tekton-results';
+import { PipelineRunModel } from '../models';
+import { useRuns } from './useRuns';
+import { getGroupVersionKindForModel } from '../utils/common-utils';
+import { FetchUtilsType } from '../types/k8s';
+
+export const usePipelineRuns = (
+  fetchUtils: FetchUtilsType,
+  namespace: string,
+  tektonResultsBaseURL: string,
+  isTektonResultEnabled: boolean,
+  options?: {
+    selector?: Selector;
+    limit?: number;
+  },
+  cacheKey?: string,
```

Comment:
I am thinking it would be better if we pack all the configuration options (fetchUtils, tektonResultsBaseURL,  and isTektonResultEnabled)  into one object called config, because this object will stay the same for the consumer (Konflux, console etc), we can define this object in the cosumer codebase and just pass it wherever we need to.



```suggestion
  namespace: string,
  config: {
       fetchUtils: FetchUtilsType,
       tektonResultsBaseURL: string,
       isTektonResultEnabled: boolean,
  },
  options?: {
    selector?: Selector;
    limit?: number;
  },
  cacheKey?: string,
```